docs/basics/environment-variables.md

```diff
@@ -43,7 +43,7 @@ For a list of Sauce Connect Proxy environment variables, see [Environment Variab
 7. For **Variable value**, enter your Sauce username.
 8. Click **OK**.
 9. Repeat 4 - 8 to set up the **SAUCE_ACCESS_KEY**.
-10. Confirm that your environment variables have been set by typing `echo $SAUCE_USERNAME` in your terminal. The response should be your username value.  Then do the same for your access key. 
+10. Confirm that your environment variables have been set by typing `echo %SAUCE_USERNAME%` in your terminal. The response should be your username value.  Then do the same for your access key. 
```

Comment:
Thanks for this correction, @abalmukund. Is the new value universal? `echo $SAUCE_USERNAME` works for me in BASH and ZSH. Why are you thinking it doesn't work? Should we instead provide a note saying that the verification command may be different if you are using a shell other than BASH or ZSH? Should we provide a few of the most common shell commands for this action? Let's think this through a bit before we just change the command, which, incidentally, is given in more places than just this file. @kimsauce @spider-sauce